app/views/details/updateAddress.scala.html

```diff
@@ -18,17 +18,23 @@
 @import helpers._
 @import controllers.manageDetails.UpdateDetailsVM
 @import config.ApplicationConfig
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
 
-@this(mainLayout: views.html.mainLayout, govukButton: GovukButton, govukInput: GovukInput, formWithCSRF: FormWithCSRF)
+@this(mainLayout: views.html.mainLayout, govukButton: GovukButton, govukInput: GovukInput, formWithCSRF: FormWithCSRF, govukErrorSummary : GovukErrorSummary)
 
 @(model: UpdateDetailsVM)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
 
 @mainLayout(
     pageTitle = Messages("updateAddress.title"),
     backUri = Some(controllers.manageDetails.routes.ViewDetails.show.url)
 ) {
-    <h1 class="govuk-heading-l">@messages("updateAddress.title")</h1>
 
+    @if(model.form.errors.nonEmpty) {
+        @govukErrorSummary(ErrorSummary(errorList = model.form.errors.asTextErrorLinks, title = Text(messages("errors.heading.title"))))
+    }
```

Comment:
I think this code block is repeated 8 times.

Reply:
`/views/helpers/errorSummary.sala.html` is probably not used anymore (you can check) - that's probably where this code should be.

Reply:
I think errorSummary.sala.html is old one with old styles, I am creating new errorSummay file with simple